samples/Console-GizyckoBot/main.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import asyncio
+from botbuilder.core import TurnContext, ConversationState, UserState, MemoryStorage
+from botbuilder.schema import ActivityTypes
+
+from adapter import ConsoleAdapter
+
+import urllib3
+from time import strftime
+import json
+
+# Create adapter
+adapter = ConsoleAdapter()
+
+# Create MemoryStorage, UserState and ConversationState
+memory = MemoryStorage()
+# Commented out user_state because it's not being used.
+# user_state = UserState(memory)
+conversation_state = ConversationState(memory)
+
+# Register both State middleware on the adapter.
+# Commented out user_state because it's not being used.
+# adapter.use(user_state)
+adapter.use(conversation_state)
+
+def is_bridge_opened():
+    time = strftime('%H')+strftime('%M')
+    if 0000 <= time >= 1030 or 1100 <= time >= 1200 or 1300 <= time >= 1330 or 1430 <= time >= 1600 or 1730 <= time >= 2359:
+        return True
+    else:
+        return False
+
+
+class Weather():
+    http = urllib3.PoolManager()
+
+    def get_weather(self):
+        r = self.http.request('GET', 'http://api.openweathermap.org/data/2.5/weather?q=Gizycko,pl&APPID=0ef268648751f483a62fd90feb04c5eb&units=metric')
+        r = json.loads(r.data.decode('utf-8'))
+        self.status = r["weather"][0]["description"]
+        self.temp = int(r["main"]["temp"])
+        self.wind_speed = r["wind"]["speed"]
```

Comment:
You're storing the retrieved data on the class itself, but without further usage of this data, it would be better to return the resulting formatted data. Looking at your code it creates a new `Weather` instance on every turn, so the data is never reused.

Additionally, it's generally frowned (in multiple languages) upon to create class properties or methods without declaring them first in the initializer or constructor.